src/molecules/formfields/CreditCardField/credit_card_field.module.scss

```diff
@@ -0,0 +1,48 @@
+@import 'assets/stylesheets/global';
+
+$form_border: color('neutral-5');
+$padding: ru(.5);
+
+.credit-card {
+  // Place your component specific scss here.
+
+  &-header {
+    display: flex;
+    align-items: center;
+    padding-right: $padding;
+  }
+
+  &-lock {
+    width: ru(.75);
+    height: ru(.75);
+    margin-left: auto;
+  }
+
+  &-line1 {
+    position: relative;
+
+    border-bottom: 1px solid $form_border;
+  }
+
+  &-number {
+    width: 100%;
+  }
+
+  &-logo {
+    position: absolute;
+    top: 0;
+    right: $padding;
+    height: 100%;
+    pointer-events: none;
+  }
+
+  &-line2 {
+    display: flex;
+  }
+
+  &-exp, &-sec { width: 50%; }
+
+  &-exp {
+    border-right: 1px solid $form_border;
```

Comment:
@patrickkim You might as well extract the whole `1px solid #color` into the variable.